### ISSUE TITLE
onie installation failure 

### ISSUE THREAD

**Issue by lguohan** [Azure/sonic-buildimage#630]
I see this once. If I reboot onie and re-install the same image, it succeeds. This is on Dell S6000 box.

```
ONIE:/ # onie-nos-install http://100.127.20.23/installer/ACS/sonic-broadcom-lgh.
bin
discover: installer mode detected.
Stopping: discover... done.
Notice:  Invalid TLV header found.  Using default contents.
Notice:  Invalid TLV checksum found.  Using default contents.
ONIE: Unable to find 'Base MAC Address' TLV in EEPROM data.
Notice:  Invalid TLV header found.  Using default contents.
Notice:  Invalid TLV checksum found.  Using default contents.
ONIE: Unable to find 'Serial Number' TLV in EEPROM data.
Info: Fetching http://100.127.20.23/installer/ACS/sonic-broadcom-lgh.bin ...
Connecting to 100.127.20.23 (100.127.20.23:80)
installer            100% |*******************************|   375M  0:00:00 ETA
ONIE: Executing installer: http://100.127.20.23/installer/ACS/sonic-broadcom-lgh.bin
Verifying image checksum ... OK.
Preparing image archive ... OK.
ONIE Installer: platform: x86_64-broadcom-r0
onie_platform: x86_64-dell_s6000_s1220-r0
Installing SONiC in ONIE
Replace ONIE reboot with Dell reset commands
deleting partition 3 ...
Warning: The kernel is still using the old partition table.
The new table will be used at the next reboot.
The operation has completed successfully.
Error: Partition(s) 3 on /dev/sda have been written, but we have been unable to inform the kernel of the change, probably because it/they are in use.  As a result, the old partition(s) will remain in use.  You should reboot now before making further changes.
Failure: Unable to install image: http://100.127.20.23/installer/ACS/sonic-broadcom-lgh.bin
```

**Comment by jleveque** [comment]
I have also encountered this a number of times on the S6000.

**Comment by lguohan** [comment]
@qiluo-msft , do you know the error message?

**Comment by qiluo-msft** [comment]
May I ask some question to check the repro conditions?
1. Only happens if there is a SONiC already installed?
2. Always happens if installed, and disappear after reboot?

**Comment by lguohan** [comment]
addressed in #1575
